docs/pm/methods/jtbd-questioning.md

```
@@ -0,0 +1,260 @@
+
+## The Four Question Categories
+
+Use 3-5 questions total. Pick the most relevant from each category based on the feature.
+
+### 1. Job Identification Questions
+
+**Purpose:** Understand what job the user is trying to get done
+
+**Questions to ask:**
+
+**"What job is the user trying to get done?"**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)